src/playlist-selectors.js

```diff
@@ -296,7 +323,9 @@ export let simpleSelector = function(
   if (chosenRep && chosenRep.playlist) {
     let type = 'sortedPlaylistReps';
 
-    if (resolutionPlusOneRep) {
+    if (leastPixelDiffRep) {
+      type = 'leastPixelDiffRep';
```

Comment:
Since we call it `leastPixelDiffRep` should we name the property `experimentalLeastPixelDiffSelector`?